### PR TITLE
Create layout

### PR DESCRIPTION
**Pull request by Greencat28**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -4,15 +4,256 @@
...
+        android:text="Samsung Health"
+        android:textColor="@color/black"
+        android:textSize="40dp"
+        app:layout_constraintBottom_toTopOf="@id/first_screen"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
+        android:id="@+id/logo_2"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: rename hint

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -4,15 +4,256 @@
...
+        android:layout_width="10dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="15dp"
+        android:src="@drawable/three_dots"
+        app:layout_constraintBottom_toTopOf="@+id/first_screen"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/logo_2" />
+
+    <android.support.constraint.ConstraintLayout
+        android:id="@+id/first_screen"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: conteiner_steps

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -4,15 +4,256 @@
...
+        android:layout_width="0dp"
+        android:layout_height="125dp"
+        android:layout_margin="10dp"
+        android:background="@drawable/corners"
+        android:padding="20dp"
+        app:layout_constraintBottom_toTopOf="@id/second_screen"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:context=".MainActivity">
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: delete

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -4,15 +4,256 @@
...
+
+        <TextView
+            android:id="@+id/steps_1"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: steps_title

File: app/src/main/res/drawable-v24/corners.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <solid android:color="#fff" />
+
+    <corners
+        android:radius="20dp"
```
Comment (CodOndinnonk) [inline app/src/main/res/drawable-v24/corners.xml]: dimens

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -4,15 +4,312 @@
...
+            style="?android:attr/progressBarStyleHorizontal"
+            android:layout_width="@dimen/steps_progressbar_width"
+            android:layout_height="wrap_content"
+            android:background="@drawable/corners"
+            android:progress="2"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="@id/indents_of_progressbar"
+            app:layout_constraintTop_toTopOf="parent"
+
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: +

File: app/src/main/res/values/dimens.xml [hunk 7]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="main_title_size">40sp</dimen>
+    <dimen name="main_hint_size">14sp</dimen>
+    <dimen name="three_dots_width">10dp</dimen>
+    <dimen name="container_width">0dp</dimen>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/dimens.xml]: -

File: app/src/main/res/values/dimens.xml [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    <dimen name="container_height">125dp</dimen>
+    <dimen name="container_padding">20dp</dimen>
+    <dimen name="container_margin_bottom">10dp</dimen>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/dimens.xml]: make universal (remove gidelines)

File: app/src/main/res/values/dimens.xml [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    <dimen name="steps_title_size">18sp</dimen>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/dimens.xml]: container_title_size

File: app/src/main/res/values/dimens.xml [hunk 10]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="main_title_size">40sp</dimen>
+    <dimen name="main_hint_size">14sp</dimen>
+    <dimen name="three_dots_width">10dp</dimen>
+    <dimen name="container_width">0dp</dimen>
+    <dimen name="container_height">125dp</dimen>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/dimens.xml]: wrap_content for containers

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -1,3 +1,16 @@
 <resources>
     <string name="app_name">Health</string>
+    <string name="main_title">Samsung Health</string>
+    <string name="main_hint">Синхронизировано с Gear S3 в 01:53</string>
+    <string name="steps_title">Шаги</string>
+    <string name="steps_current">34</string>
+    <string name="steps_goal">/9 000</string>
+    <string name="steps_hint">Все шаги, 02:14</string>
+    <string name="steps_percent">0%</string>
+    <string name="three_dots">three_dots</string>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/strings.xml]: ?

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -1,3 +1,16 @@
...
+    <string name="exercise_title">Упражнения</string>
+    <string name="exercise_journal">Посмотреть журнал</string>
+    <string name="sleep_title">Вы спали?</string>
+    <string name="sleep_enter">OK</string>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/strings.xml]: ok

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (Greencat28) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (Greencat28) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (Greencat28) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (Greencat28) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/drawable-v24/corners.xml (hunk 5 above)
Comment (Greencat28) [reply app/src/main/res/drawable-v24/corners.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 6 above)
Comment (Greencat28) [reply app/src/main/res/layout/activity_main.xml]: Deleted extra space on 177 line. Ok?

File: app/src/main/res/values/dimens.xml (hunk 7 above)
Comment (Greencat28) [reply app/src/main/res/values/dimens.xml]: Returned 0dp in XML

File: app/src/main/res/values/dimens.xml (hunk 8 above)
Comment (Greencat28) [reply app/src/main/res/values/dimens.xml]: done

File: app/src/main/res/values/dimens.xml (hunk 9 above)
Comment (Greencat28) [reply app/src/main/res/values/dimens.xml]: done (huge, big, normal, little size text)

File: app/src/main/res/values/dimens.xml (hunk 10 above)
Comment (Greencat28) [reply app/src/main/res/values/dimens.xml]: Disagree. Then I have trouble with bias and etc.

File: app/src/main/res/values/strings.xml (hunk 11 above)
Comment (Greencat28) [reply app/src/main/res/values/strings.xml]: done (delete)

File: app/src/main/res/values/strings.xml (hunk 12 above)
Comment (Greencat28) [reply app/src/main/res/values/strings.xml]: done (renamed)

File: app/src/main/res/layout/activity_main.xml [hunk 13]
```diff
@@ -1,18 +1,282 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/binding_for_main_title" />
 
     <TextView
+        android:id="@+id/main_hint"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/main_hint"
+       android:textColor="@color/black"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: не критично, но не сделала выравнивание

Comment (Greencat28) [reply app/src/main/res/layout/activity_main.xml]: done

**Comment by CodOndinnonk** [comment]
Created layout

**Pull request merged**